skimage/filters/tests/test_thresholding.py

```diff
@@ -426,3 +428,22 @@ def test_niblack_sauvola_pathological_image():
     value = 0.03082192 + 2.19178082e-09
     src_img = np.full((4, 4), value).astype(np.float64)
     assert not np.any(np.isnan(threshold_niblack(src_img)))
+
+
+def test_bimodal_multiotsu_hist():
+    image = data.camera()
+    thr_otsu = threshold_otsu(image)
+    thr_multi, _ = threshold_multiotsu(image, classes=2)
+    assert thr_otsu == thr_multi
+
+
+def test_check_multiotsu_results():
+    image = 0.25 * np.array([[0, 1, 2, 3, 4],
+                             [0, 1, 2, 3, 4],
+                             [0, 1, 2, 3, 4],
+                             [0, 1, 2, 3, 4]])
+
+    for idx in range(3, 6):
+        thr_multi, _ = threshold_multiotsu(image,
```

Comment:
it's ``thr_multi = ....`` now that the function only returns the thresholds (this causes Travis to fail)

Reply:
Thanks! Didn't see that 🤔😳